webui/src/lib/components/auth/layout.tsx

```diff
@@ -23,6 +27,15 @@ export const AuthLayout = () => {
     return (
         <Container fluid="xl">
             <Row className="mt-5">
+                <div>
+                    {rbac === 'simplified' &&  showRBACAlert &&
+                    <Alert severity="info" title="rbac CTA" dismissible onClose={() => {
+                        window.localStorage.setItem(rbacDismissedKey, "true");
+                        setShowRBACAlert(false);
+                    }}><InfoIcon/>{" "}Enhance Your Security with {" "}<Alert.Link href={"https://docs.lakefs.io/reference/security/rbac.html"}>Role-Based Access Control</Alert.Link>{" "}
+                        – Available on <Alert.Link href={"https://lakefs.cloud/register"}>lakeFS Cloud</Alert.Link> and <Alert.Link href={"https://docs.lakefs.io/understand/enterprise/"}>lakeFS Enterprise</Alert.Link>!</Alert>
+                    }
+                </div>
```

Comment:
Neat!  But consider removing that div, when dismissed:
```suggestion
                    {rbac === 'simplified' &&  showRBACAlert &&
                    <div><Alert severity="info" title="rbac CTA" dismissible onClose={() => {
                        window.localStorage.setItem(rbacDismissedKey, "true");
                        setShowRBACAlert(false);
                    }}><InfoIcon/>{" "}Enhance Your Security with {" "}<Alert.Link href={"https://docs.lakefs.io/reference/security/rbac.html"}>Role-Based Access Control</Alert.Link>{" "}
                        – Available on <Alert.Link href={"https://lakefs.cloud/register"}>lakeFS Cloud</Alert.Link> and <Alert.Link href={"https://docs.lakefs.io/understand/enterprise/"}>lakeFS Enterprise</Alert.Link>!</Alert></div>
                    }
```